.github/workflows/java.yaml

```diff
@@ -9,6 +9,29 @@ on:
     - cron: '0 0 * * *'
 
 jobs:
+  check-copyright:
+    name: Check Copyright
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Check Copyright
+        env:
+          EXTENSIONS: "*.java"
```

Comment:
please could you change the EXTENSIONS env variable to:
EXTENSIONS: "*.java *.js"